domain/src/main/kotlin/no/nav/su/se/bakover/domain/brev/BrevbestillingId.kt

```diff
@@ -1,8 +1,9 @@
 package no.nav.su.se.bakover.domain.brev
 
+import com.fasterxml.jackson.annotation.JsonCreator
 import com.fasterxml.jackson.annotation.JsonValue
 
-data class BrevbestillingId(val value: String) {
+data class BrevbestillingId @JsonCreator(mode = JsonCreator.Mode.DELEGATING) constructor(val value: String) {
```

Comment:
Støtt gjerne opp noe slikt med noen simple tester. Kjekt verktøy for å finne ut om man faktisk endrer på oppførselen også.